pr2eus/robot-interface.l

```diff
@@ -81,7 +81,12 @@
           (setq current-time 0)
           )
         av)))
-  (:interpolatingp () (not (null timer-sequence)))
+  (:interpolatingp
+   ()
+   (cond ((send ri :simulation-modep)
+          (not (null timer-sequence)))
+         (t ;; real robot
+          (eq (send self :get-state) ros::*simple-goal-state-active*))))
```

Comment:
According to my review at #351, how about changing this line to:

`(not (eq (send self :get-state) ros::*simple-goal-state-done*))`?

c.f. https://github.com/jsk-ros-pkg/jsk_roseus/blob/master/roseus/euslisp/actionlib.l#L203

Reply:
updated, note that `get self :get-state` do not return `ros::*simple-goal-state-active*`, need to check `ros::simple-state`